app/com/arpnetworking/metrics/portal/reports/impl/EmailSender.java

```diff
@@ -32,12 +50,78 @@
 public class EmailSender implements Sender {
     @Override
     public CompletionStage<Void> send(
+            final Report report,
             final Recipient recipient,
-            final ImmutableMap<ReportFormat, RenderedReport> formatsToSend
+            final ImmutableMap<ReportFormat, RenderedReport> formatsToSend,
+            final Instant scheduled
     ) {
-        throw new RuntimeException();
+        return CompletableFuture.supplyAsync(() -> {
+            try {
+                sendSync(report, recipient, formatsToSend, scheduled);
+            } catch (final IOException e) {
+                throw new CompletionException(e);
+            }
+            return null;
+        });
     }
 
-    EmailSender() {}
+    private void sendSync(
+            final Report report,
+            final Recipient recipient,
+            final ImmutableMap<ReportFormat, RenderedReport> formatsToSend,
+            final Instant scheduled
+    ) throws IOException {
+
+        final String subject = getSubject(report, scheduled);
+        EmailPopulatingBuilder builder = EmailBuilder.startingBlank()
+                .from(_fromAddress)
+                .to(recipient.getAddress())
+                .withSubject(subject);
+
+        for (final RenderedReport rendered : formatsToSend.values()) {
+            builder = addFormat(builder, rendered);
+        }
+
+        LOGGER.info()
+                .setMessage("sending email")
+                .addData("report", report)
+                .addData("recipient", recipient)
+                .log();
+
+        _mailer.sendMail(builder.buildEmail());
+    }
+
+    private String getSubject(final Report report, final Instant scheduled) {
+        final String formattedTime = ZonedDateTime.ofInstant(scheduled, ZoneOffset.UTC).toString();
```

Comment:
I feel weird about using `UTC` here. Perhaps we need to promote timezone from being a `Schedule`-level concept to being a field on `Report`? (And maybe `Report` should eventually contain a lot of locale/UI information?)

If we're playing this on hard mode, though: suppose a report is being sent to some people in SFO and some people in NYC. What should the timestamp in the subject line be _then_?

(Or maybe we should just punt on this until we have configurable/templatable subject lines.)